setup.py

```diff
@@ -33,7 +33,7 @@
         # License: BSD
         # Upstream url: https://github.com/pyca/cryptography
         # Use: For encryption
-        'cryptography>=1.2.1,<3.0.0',
+        'cryptography>=1.2.1,<3.4.8',
```

Comment:
Thought I had commented before but maybe it was lost. Any specific reason this is `<3.4.8`? Can it be `<4.0` or a more recent major version that we've tested against?

Reply:
https://github.com/pyca/cryptography/tags
I think the latest correct one is 3.4.8, they mess up the latest one to be 35.0.0
That's why I pick up 3.4.8
Any suggestions here?

Reply:
Bump the upper boundary to 4.0.0, which include the 3.4.8 also.  As long as version >3.1 will be good